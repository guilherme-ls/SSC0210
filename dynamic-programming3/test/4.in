5
wehdmarhyfcknwrgjisjbagmxfkiekmhrtqfuxrwfduenfzqvyekoubqpjwjbvmffkhqurphwkjyedwlzvhwtjgrbzjjonsnnwagdqtxknphjomummhcooigrqownxopiivftuipaefhhhdzvhcvdhuuowgfewbqnkudefggknobmaeudwceuitykroocylhmhvwwvyphiqwxtzfpzzftmmntfxpodzhcs
fkbhawypghinrrexzfbfwazmmfrzzmuhtcssjcbaeaaqhnepklkcvnftbiwfmigduncnduzeczcgrcynkoliiqqnwfgsrwfbxefcubjzdevvhvjhjabbcbiwtorgfdemrtxsqejutxoahkjkcqjhwnrfdxakjyfhdhpmnqclvzmxwedteppedpzfaarkyoxqorzawobkqqbworumoltieavauihzzsahufkxfzqnbyhtyplhvexobhbocwggqvqcfqvvrpqxmhmimkuypdowzptxrzynlydmmibgayoaiaijrkmxfpxiqrtaggrvxiso
uphzuuqhbstzineucxzcbyxbhmdiuohvpqmwtvwifpgqhnyqoxoatzcxkvcmmtxhtapbcdcbpiibgicxyewvkdoowbolwmbnmpikdzauozvcwfykpaitqplblearkapfybnycnrktlchgwpsfqdsqyryjneumnqhuapmrccjyigeyluuqgwpaszkhnpgswdvaiarutdsendlybepxtpyxtopzqpvwvysnpudcnrmqbjgwycigogufjsvykwxgarguqepfaftkhhlbzdetugndvqzkssohimqdmfyylhhjtqcgitnqohkfemytibtltxjsobylrtmwqqxhresuapyosvzrtmxiszzygunkiirotuqyrushcrloelpxaotgattrbdafwqdvsouwalfnbvyzvyytdmdrrdyfwkcliqnljdoqlqponwstihngfiyvcjojdememurmxtkaudib
hkbzwufzhbryihbwaypjxqlsergbssmkquihnboiygapkuxzontbslwsayznbphhuesvmmnaoehvgejchsjemipejukjwnxzzcmepsopajqtbmatpldakbdhiqfuhrkfwtctskyavvhfxketovxgykojizranvjpqoptfqgygphqeynlbwdrbltjmwmmjetiatdmujrunabrmhgjwjogsablndizhdjrbtgyrbmrswllzahkswyyssyrewciaiodpmzmsqqqtkcfsmiamwtcgtuchlbutsejyjfeavbhyzwwhykbebhgptsvrwzinfzdnegxddhsqeyziygmiapensllssmrziasorlfjjihrofzggkgxytcgmaplwlxhyhqguybiabmrbtrspurivuhslouyuzwbnsjjclagmcscjviprxhsikasckseslhiticsbfzaybybnwlcnuuiwydlnli
frglyddsamnzckouxxawuqtsmqjvpbjjynpbncogdunnwzoqdvfkwcwfxhxmwxkbierxqdulwhsnkvwjsbipvshahedymnvkvozrxfqdgnqmzqypphhubtwxstfvngstjmlmoadrlhwnrtlmwtvojrshtnlewxeryoqkrlwouzuozwsxcilhzvnfsprigzjzuthpjddgaxiyykinqdcctstwtvprqnzbvrpegezdevuuqnfyfefmbchjytqcihwdzmmmuslvgwbuyvrpakmxajofuas
hqbaspwtdsrrzusvqkzgwfmpdwpjfzeuqmumtqkttwunhvczvhjoxjnieteaciqtxgkrqbcijgjsrfbjuukcxgubbmpyajsnundhamrxneswoxdshgtyfpykebgewnljjhjfumqnfslgjbqkdgftrefmhffmhfeqwmbxwxzsxsnjqjonlehfnrpgxtvdxhlddptkfvjdyzjubvjwqldxqrqsuxlxxygigzzcsvhpwctbpkgrmxqiiojjzbfbqlhkzbqypipruuplqrdizenqfvqevoedecgtvqemxfelpkpaeuqoqvdpqiiryxnscrkafmgrbmiyqfogpwmljumpryqcjnfglckxpgfpbhidxkgmrhcsufmqiabgqmeljedfyosyugdijyaqlmosfjavotggbelkvqazrrsytgkynqlzq
gifztdqvixohoatqmbfhgutrayrhmaifoduehufaqbbn
sprnleemxtagkpmajmbparjohnmy
zantzefexghzvszmpcaroiumeofpobdkwibmsircxrehxbunqpcjizrnxknxwfzindfujutwnopuphribobukawivxfgvrmwrccuxjzlbznoitmjcgppidukroymgsuogtjdfxkijoqqfdxbuvhvjjbulkjllwizfdqkfdqyczizxxpbbnljopcpgytmjoxxqwjyfivmjpzdiibcoxwbsjrsopmshdrlpsuworcbrbtqhymuksumudecpmfotwyhxkibfeumifyhoymczbcwwxlpkyvqquxqtawibizvlhssefewylnkjo
eatrmwfpiusnszimkwbyrcpvmcrwuzjliirhygnnojifjspjrzmaecrrvcjmosphboeiysrjyhxzlmasddnbuorpfgzmksekxvrdwvgrlplskxppyqftqprjmilzayvrvmfpfgnutbfljjymouexjpnpcyuh
