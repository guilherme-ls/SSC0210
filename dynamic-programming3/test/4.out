261
413
353
37
252
